nvi-rest/src/test/java/no/sikt/nva/nvi/rest/fetch/FetchNviCandidateByPublicationIdHandlerTest.java

```diff
@@ -84,6 +88,19 @@ void shouldReturnValidCandidateWhenCandidateExists() throws IOException {
         assertEquals(expectedResponse, actualResponse);
     }
 
+    @Test
+    void shouldReturnCandidateWithReportStatus() throws IOException {
+        var candidate = candidateRepository.create(randomCandidate().copy().reportStatus(ReportStatus.REPORTED).build(),
+                                                   List.of(randomApproval()));
+        var request = requestWithAccessRight(candidate.candidate().publicationId());
+
+        handler.handleRequest(request, output, CONTEXT);
+        var response = GatewayResponse.fromOutputStream(output, CandidateDto.class);
+        var actualCandidate = response.getBodyObject(CandidateDto.class);
+
+        assertEquals(actualCandidate.reportStatus(), ReportStatus.REPORTED.getValue());
```

Comment:
Invert the condition, since you're using Junit assertions which expect method(expected, actual)